dbt_project.yml

```diff
@@ -26,9 +26,10 @@ vars:
   conversion_window_start_date: '' # conversion window start date
   conversion_window_end_date: '' # conversion window end date is dynamically set to current_date()-1 in the code
   conversion_hosts: ['poplindata.com'] # url_hosts to consider
```

Comment:
did we want to change this to 'a.com' like what is in the sample data, or is that being done somewhere else?

Reply:
First I was thinking of changing it in the accelerator instructions but good point, we have the chance to change it in the source here for future users.